src/test/java/org/javawebstack/orm/test/DatabaseConnection.java

```diff
@@ -0,0 +1,43 @@
+package org.javawebstack.orm.test;
+
+import org.javawebstack.orm.wrapper.MySQL;
+import org.javawebstack.orm.wrapper.SQL;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public abstract class DatabaseConnection {
```

Comment:
I'm not sure if an abstract class is the best way to to this, e.g. a user may use connections to multiple databases. What do you think about it?

Reply:
This is only for the test so the user won't really see it.

Reply:
Okay, but with this we are limited to this one extends from ORMTestCase

Reply:
Can you come to teamspeak to talk about this?

Reply:
He moved the sql connection logic out of the base test class (ORMTestCase) and ORMTestCase aswell as any potential future base class can extend DatabaseConnection aswell to use the MySQL Connection. A single database should be totally fine for our testing purposes. Everything in src/test is compiled but not packaged into the library, so you won't be able to use this base class in your application's tests anyway. We could introduce something like that but in an application environment you are most likely going to fire up your entire application anyway so it's logic will handle connections.